### ISSUE TITLE
Exclusive ingredients

### ISSUE THREAD

**Issue by marmistrz** [thinkle/gourmet#813]
I'd like to set up an exclusive group for one recipe. For example, one should add **either** 500 g of chicken **or** 250 g of bacon.

Is there any way to do it? If yes, it should be documented, if not - it'd be a feature request.

**Comment by thinkle** [comment]
No. I thought about implementing this but couldn't come up with a nice UI.

Currently, the way to do it would be to make *both* optional. You could put
them in a group called "Choose one" -- then it would display as

Choose one
- Bacon        [   ] (checkbox)
- Chicken     [   ] (checkbox)

That seems like a pretty good approximation to me, but it's not what you're
asking for. I guess the question I'd have for you is, what should the user
interface look like for entering these groups in, and how do you add such a
UI without cluttering up things?

We'd also have to figure out the data model, but that doesn't necessarily
have to be an enduser's problem :)

Tom

On Tue, Sep 15, 2015 at 9:02 AM, Marcin Mielniczuk <notifications@github.com

**Comment by marmistrz** [comment]
Well, I have one idea:

500 ml milk
`f(250)` g chicken `qsTr(or)` `f(500)` g bacon - `f(x)` multiplies, `qsTr` - translation. So it would look like:

500 g kurczaka **albo** 1000 g boczku
500 g chicken **or** 1000 g bacon

when multiplied by 2
